### PR TITLE
Revert batch 3 of go update.

### PR DESCRIPTION
**Pull request by mattmoor**
```
git revert 66acd7953418b2f98a18ab6c76033e8c27031281 -m 1
```

ref: https://github.com/wolfi-dev/os/pull/6132

### CODE REVIEW

**Comment by mattmoor** [comment]
This is pre-emptive.  DO NOT MERGE this unless we need it because the prior rollback is insufficient to unwedge things.

**Review by imjasonh — APPROVED** [review]

**Comment by mattmoor** [comment]
It looks like things are passing, so in all likelihood we will close this out 🤞

**Comment by mattmoor** [comment]
Ok, the upload step is running out of space.  I am going to merge this to try and make progress and then we can make a judgement call about reverting things back.

**Pull request merged**
